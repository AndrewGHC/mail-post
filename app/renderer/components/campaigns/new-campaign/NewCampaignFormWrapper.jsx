import React from 'react';
import Paper from 'material-ui/Paper';

const NewCampaign = () => (
  <Paper>
    NewCampaign
  </Paper>
);

export default NewCampaign;
