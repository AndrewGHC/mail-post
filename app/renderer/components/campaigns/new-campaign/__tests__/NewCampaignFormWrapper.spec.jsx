import React from 'react';
import { shallow } from 'enzyme';

import NewCampaignFormWrapper from '../NewCampaignFormWrapper';

describe('NewCampaignFormWrapper', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<NewCampaignFormWrapper />);
    expect(wrapper).toMatchSnapshot();
  });
});
