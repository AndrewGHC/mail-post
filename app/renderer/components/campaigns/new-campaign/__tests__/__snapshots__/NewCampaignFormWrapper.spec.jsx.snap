// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCampaignFormWrapper matches snapshot 1`] = `
<WithStyles(Paper)>
  NewCampaign
</WithStyles(Paper)>
`;
